projects/cdk/directives/auto-focus/handlers/default.handler.ts

```diff
@@ -29,7 +29,9 @@ export class TuiDefaultAutofocusHandler extends AbstractTuiAutofocusHandler {
                     skipWhile(Boolean),
                     take(1),
                 ),
-            ).subscribe(() => this.element.focus({preventScroll: true}));
+            ).subscribe(() =>
+                this.element.focus({preventScroll: this.options.preventScroll}),
+            );
         } else {
             this.element.focus({preventScroll: true});
```

Comment:
Should we add `{preventScroll: this.options.preventScroll}` for `this.element.focus({preventScroll: true});`?

Reply:
no, that would be a breaking change for the buttons